apps/src/sites/hourofcode.com/pages/views/hoc_events_map.js

```diff
@@ -54,6 +54,14 @@ $(document).ready(function () {
       url: 'mapbox://codeorg.hoctiles',
     });
 
+    map.setFilter('admin-0-boundary-disputed', [
```

Comment:
Would `admin-0-boundary` not show the faint boundary line since this layer "shows all boundaries at this level that are not disputed"? I'm not familiar at all with these filter types, just reading off of [the site](https://docs.mapbox.com/mapbox-gl-js/example/toggle-worldviews/), so I may be misunderstanding this!

Reply:
I had that thought too, but unfortunately it does the opposite (makes the Moroccan border show back up in a dotted line). I think this is classified in all worldviews as a disputed boundary.

Reply:
Also! I will update the pr description- I'm checking with Leo to make sure this view is acceptable to the stakeholder. I believe the faint border is the standard depiction of a territory line (as in, can't be removed), in contrast to the dotted line for 'disputed'.

Reply:
Oh gotcha, thank you! Then this looks perfect!